### PR TITLE
fix(webapp): fix something bugs

### LINKED ISSUE #860: Sort by Total votes
The sort by total votes in not correct, the top 21 BPs shuld be the same as on blocks.io

image

### LINKED ISSUE #867: Add my rating in bp profile and rating page

### LINKED ISSUE #870: Review all sorting
Detected error in some sort:

- Top Rated Eden
- Top Rated General
- Top alphabetical

image

### LINKED ISSUE #874: My rating data on rate page is bad

### PR DESCRIPTION
**Pull request by AngeloCG97**
## My rating data on rate page is bad

Resolve #867
Resolve #860
Resolve #870
Resolve #874

### CODE REVIEW

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by tetogomez — APPROVED** [review]
LGTM

File: hapi/src/libs/sync-stats.js [hunk 1]
```diff
@@ -23,33 +25,132 @@ const getRatingsStats = async (scope) => {
...
+              statsAmount: 5,
+              oneStat: 1
+            }),
+            bp: rating.bp
+          }
+        ]
+        break
+      }
+    }
+    if (!haveTowRating) {
```
Comment (leisterfrancisco) [inline hapi/src/libs/sync-stats.js:80]: haveTowRating -> haveTwoRating

Comment (leisterfrancisco) [reply hapi/src/libs/sync-stats.js:80]: leave space before

File: hapi/src/libs/sync-stats.js [hunk 2]
```diff
@@ -23,33 +25,132 @@ const getRatingsStats = async (scope) => {
...
+
+  edenRatingsStats.rows.forEach(rating => {
+    let haveTowRating = false
+    for (let index = 0; index < generalRatingsStats.rows.length; index++) {
+      if (rating.bp === generalRatingsStats.rows[index].bp) {
+        haveTowRating = true
+        break
+      }
+    }
+    if (!haveTowRating) {
```
Comment (leisterfrancisco) [inline hapi/src/libs/sync-stats.js:122]: add space before

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: hapi/src/config/index.js [hunk 3]
```diff
@@ -2,19 +2,19 @@ const massive = require('massive')
...
-    massiveConfig,
-    massiveDB,
-    generalContractScope: process.env.HAPI_RATING_CONTRACT || 'rateproducer',
-    edenContractScope: 'eden'
-}
\ No newline at end of file
+  massiveConfig,
+  massiveDB,
+  generalContractScope: process.env.HAPI_RATING_CONTRACT || 'rateproducer',
+  edenContractScope: 'eden'
```
Comment (leisterfrancisco) [inline hapi/src/config/index.js:19]: @AngeloCG97 there is a global variable for this, it is `edenContractScope`, try to use it

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by leisterfrancisco — APPROVED** [review]

**Pull request merged**